perf/stability/control-plane/test.sh

```diff
@@ -0,0 +1,78 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+setup() {
+  kubectl create ns twopods-istio
+  kubectl create ns invalid-namespace-0
+  kubectl label namespace invalid-namespace-0 istio-injection=enabled --overwrite
+  kubectl apply -f <(istioctl kube-inject -f ../../istio-install/tmp/istio-1.12.0/samples/sleep/sleep.yaml) -n invalid-namespace-0
+  kubectl apply -f <(istioctl kube-inject -f ../../istio-install/tmp/istio-1.12.0/samples/httpbin/httpbin.yaml) -n twopods-istio
+}
+
+checkResourcesInCluster() {
+  local status
+  status=$(kubectl exec "$(kubectl get pod -l app=sleep -n invalid-namespace-0 -o jsonpath={.items..metadata.name})" -c sleep -n invalid-namespace-0 -- curl http://httpbin.twopods-istio:8000/ip -sS -o /dev/null -w "%{http_code}\n")
+  if [ "$status" -ne 403 ]
+  then
+    echo "Deny policies are not deployed properly"
+    exit 1
+  fi
+}
+
+addPolicy() {
+  go run ../../benchmark/security/generate_policies/generate_policies.go ../../benchmark/security/generate_policies/generate.go ../../benchmark/security/generate_policies/jwt.go -configFile="config.json" > deployment.yaml
+  kubectl apply -f deployment.yaml
+  checkResourcesInCluster
+  
+  
+  go run ../../benchmark/security/generate_policies/generate_policies.go ../../benchmark/security/generate_policies/generate.go ../../benchmark/security/generate_policies/jwt.go -configFile="config_100.json" > deployment_100.yaml
+  kubectl apply -f deployment_100.yaml
+  checkResourcesInCluster
+  
+
+  go run ../../benchmark/security/generate_policies/generate_policies.go ../../benchmark/security/generate_policies/generate.go ../../benchmark/security/generate_policies/jwt.go -configFile="config_1000.json" > deployment_1000.yaml
+  kubectl apply -f deployment_1000.yaml
+  checkResourcesInCluster
+}
+
+updatePolicy() {
+  kubectl delete AuthorizationPolicy --all -n twopods-istio
+
+  # update the number of namespaces on which authz policy applies
+  go run ../../benchmark/security/generate_policies/generate_policies.go ../../benchmark/security/generate_policies/generate.go ../../benchmark/security/generate_policies/jwt.go -configFile="config_update_ns.json" > deployment_update_ns.yaml
+  checkResourcesInCluster
+
+  kubectl delete AuthorizationPolicy --all -n twopods-istio
+
+  # update the number of paths on which authz policy applies
+  go run ../../benchmark/security/generate_policies/generate_policies.go ../../benchmark/security/generate_policies/generate.go ../../benchmark/security/generate_policies/jwt.go -configFile="config_update_paths.json" > deployment_update_paths.yaml
+  checkResourcesInCluster
+}
+
+
+cleanCluster() {
+  kubectl delete ns twopods-istio
+  kubectl delete ns invalid-namespace-0
+}
+
+setup
+addPolicy
```

Comment:
this just add/update the policy once, could we add a loop to keep repeating the process for a longer period?

Reply:
sure, I will do that.

Reply:
is it okay if I run it for 10 times? One single iteration runs for around 25-30mins